api/routes/routes.go

```diff
@@ -20,6 +20,8 @@ func InitRoutes(controllers Controllers, c *chi.Mux) {
 		r.Use(middleware.AllowContentType("application/json"))
 
 		r.Route("/pets", func(r chi.Router) {
+			r.Get("/{page}", controllers.PetController.ListAllPets)
```

Comment:
pq temos duas rotas para o mesmo controller?
Uma rota so que aceita uma pagina, e se for null pega a primeira :)

Reply:
alterado para uma única rota  ✅